src/util/url-utils.ts

```diff
@@ -40,3 +42,27 @@ export const parse = (header: string): any => {
   });
   return links;
 };
+
+/**
+ * Retrieve new data when infinite scrolling
+ * @param currentData
+ * @param incomingData
+ * @param links
+ */
+export const loadMoreData = (currentData: any, incomingData: any, links: any): any => {
```

Comment:
can we rename it to `loadMoreDataWhenScrolled`

Reply:
Also, can we rename the `parse` method you added here to `parseHeaderForLinks`

Reply:
maybe we should move loadMoreDataWhenScrolled move to another more appropriate file ?